gcp/inference/handler.py

```diff
@@ -19,11 +19,12 @@
     scale_coords,
 )
 
-sys.path.insert(0, "/home/model-server")
-
 from street2sat_utils.client import Prediction
 from street2sat_utils.constants import CROP_CLASSES
 
+sys.path.insert(0, "/home/model-server")
```

Comment:
I think this particular line actually needs to stay in the original location to make below imports possible. I suggest adding a `# noqa` comment at the end of the line to disable the flake8 check here